tests/integration/values/values_helm_traces_enabled.yaml

```diff
@@ -0,0 +1,68 @@
+sumologic:
+  setupEnabled: true
+  accessId: "dummy"
+  accessKey: "dummy"
+  endpoint: http://receiver-mock.receiver-mock:3000/terraform/api/
+
+  traces:
+    enabled: true
+
+# Prevent snowball effect by filtering out receiver mock logs
+fluent-bit:
```

Comment:
Why do we set this (and others like Prometheus or Fluentd) for tracing tests?

Reply:
I think we should include `sumologic.traces.enabled: true` and the `otelcol` and `otelagent` sections only 🤔

Reply:
This is to prevent fluent-bit ingesting receiver-mock logs, which would then be printed out on stdout and be ingested by fluent-bit and so on.

We could either keep this filter or disable logs for this particular testcase.

I'm open to suggestions but I thought that we should check that everything seems to work fine when we change a different config option like `traces.enabled`.

Reply:
> Why do we set this (and others like Prometheus or Fluentd) for tracing tests?

There are comments explaining why we need them :) TLDR, with default requests those pods wouldn't fit on the github runner's node